package.json

```diff
@@ -52,6 +52,7 @@
     "@sasjs/lint": "1.11.2",
     "@sasjs/utils": "2.45.0",
     "adm-zip": "0.5.9",
+    "axios": "0.27.2",
```

Comment:
You don't need to add `axios` as a dependency because it is already there (brought by `@sasjs/adapter`)